iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -244,6 +245,26 @@ void updateUser(EventProcessingRequest request) throws IOException {
         }
     }
 
+    private Map<String, Object> getUserAttributes(EventProcessingRequest request) {
+        String settingValue = request.getAccount().getAccountSettings().get(SETTING_PARSE_USER_ATTRIBUTES);
+        Boolean parseUserAttributes = Boolean.parseBoolean(settingValue);
```

Comment:
```suggestion
        boolean parseUserAttributes = Boolean.parseBoolean(settingValue);
```

nit ^^ no reason to "autobox" here, `parseBoolean` returns a primitive